docs/tutorials/quickstart-demo.md

```diff
@@ -1,28 +1,28 @@
 # Ambassador Tutorial
 
-In this tutorial, we'll walk through some of the key features and workflow of Ambassador.
+In this article, you will explore some of the key features of the Ambassador
+Edge Stack by walking through an example workflow and exploring the 
+Edge Policy Console.
 
-## Create a Mapping
+## Prerequisites
 
-In a typical configuration workflow, Custom Resource Definitions (CRDs) are used to define the intended behavior of Ambassador Edge Stack. In this demo, we'll deploy a sample service and create a `Mapping` resource. Mappings allow you to associate parts of your domain with different URLs, IP addresses, or prefixes.
+You must have [Ambassador Edge Stack installed](../getting-started/) in your 
+Kubernetes cluster.
 
-1. We'll start by deploying the `quote` service. Save the below configuration into a file named `quote.yaml`. This is a basic configuration that tells Kubernetes to deploy the `quote` container and create a Kubernetes `service` that points to the `quote` container.
+## Routing Traffic from the Edge
+
+Like any other Kubernetes object, Custom Resource Definitions (CRDs) are used to
```

Comment:
Maybe ditch `Like any other Kubernetes object, ` ? I'm not really sure what the point of it is, here.